user/urls.py

```diff
@@ -1,9 +1,15 @@
 from django.urls import path
-from .views import UserListCreateView, UserRetrieveDeleteView, SendOTPView, VerifyOTPView
-
+from .views import (
+    UserListCreateView,
+    UserRetrieveDeleteView,
+    GetCurrentUserView,
+    SendOTPView,
+    VerifyOTPView
+)
 
 urlpatterns = [
     path('', UserListCreateView.as_view(), name='list_and_create'),
+    path('me/', GetCurrentUserView.as_view(), name='get_current_user'),
```

Comment:
Couldn't think of a better endpoint than "me". Disappointed in you really

Reply:
Glad to disappoint you. 😇